app/src/main/res/layout-land/admin_auth_activity.xml

```diff
@@ -48,105 +49,81 @@
 
       <ScrollView
         android:id="@+id/admin_auth_landscape_scrollView"
-        android:scrollbars="none"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="match_parent"
+        android:scrollbars="none">
 
         <androidx.constraintlayout.widget.ConstraintLayout
           android:layout_width="match_parent"
           android:layout_height="wrap_content">
 
-          <TextView
-            android:id="@+id/heading_text"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="40dp"
-            android:fontFamily="sans-serif-medium"
-            android:lines="2"
-            android:paddingStart="28dp"
-            android:paddingEnd="28dp"
-            android:text="@string/admin_auth_heading"
-            android:textColor="@color/oppiaPrimaryText"
-            android:textSize="20sp"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent" />
+          <Button
+            android:id="@+id/submit_button"
+            style="@style/StateButtonActive"
+            android:layout_marginTop="@dimen/submit_margin_top_bottom"
+            android:layout_marginBottom="@dimen/submit_margin_top_bottom"
+            android:background="@{viewModel.isSubmitButtonActive ? @drawable/state_button_primary_background :@drawable/start_button_transparent_background}"
+            android:minHeight="@dimen/min_height"
+            android:text="@string/state_continue_submit"
```

Comment:
Create separate string for this button with label `admin_auth_submit`. Avoid using strings that relates to some other files. Make necessary changes where ever required.

Reply:
ok